### PR TITLE
Added JSX example to readme

### PR DESCRIPTION
**Pull request by maraisr**

### CODE REVIEW

**Comment by maraisr** [comment]
Fixes #6

**Comment by jbucaran** [comment]
@maraisr Man, this JSX looks so sexy! 💄

File: README.md [hunk 1]
```diff
@@ -218,6 +218,30 @@ app({ model, view, update })
 [View online](http://codepen.io/jbucaran/pen/zNxRLy?editors=0010)
 </details>
 
+<details>
+<summary>JSX</summary>
+
+```jsx
+const view = (model, msg) => (
+    <div>
+        <button onclick={msg.add}>Add</button>
```
Comment (selfup) [inline README.md]: is `onclick` valid in JSX? I thought it was `onClick`

If not, ignore my comment 😂

File: README.md [hunk 2]
```diff
@@ -218,6 +218,30 @@ app({ model, view, update })
...
+        <h1>{model}</h1>
+        <button onclick={msg.sub}>Subtract</button>
```
Comment (selfup) [inline README.md]: The original example has a disabled html property that turns to true when the number is <= 0

Awesome work though!

File: README.md (hunk 1 above)
Comment (maraisr) [inline README.md]: Yeah man! haha my bad!

**Comment by maraisr** [comment]
Mind you guys, this example currently doesn render correctly. It stops at the button.. Ill investigate and update my jsx branch to include the mods that are needed.

File: README.md (hunk 1 above)
Comment (selfup) [inline README.md]: No worries!

**Pull request closed without merging**